mmcv/image/io.py

```diff
@@ -13,15 +13,54 @@
     from cv2 import CV_LOAD_IMAGE_COLOR as IMREAD_COLOR
     from cv2 import CV_LOAD_IMAGE_GRAYSCALE as IMREAD_GRAYSCALE
     from cv2 import CV_LOAD_IMAGE_UNCHANGED as IMREAD_UNCHANGED
+try:
+    from turbojpeg import TJCS_RGB, TJPF_BGR, TJPF_GRAY, TurboJPEG
+except ImportError:
+    TJCS_RGB = TJPF_GRAY = TJPF_BGR = TurboJPEG = None
+
+jpeg = None
+supported_backends = ['cv2', 'turbojpeg']
 
 imread_flags = {
     'color': IMREAD_COLOR,
     'grayscale': IMREAD_GRAYSCALE,
     'unchanged': IMREAD_UNCHANGED
 }
```

Comment:
```python
imread_flags = {
    'cv2': {
        'color': IMREAD_COLOR,
        'grayscale': IMREAD_GRAYSCALE,
        'unchanged': IMREAD_UNCHANGED
    },
    'turbojpeg': {
        'rgb': TJCS_RGB,
        'bgr': TJPF_BGR,
        'grayscale': TJPF_GRAY
    }
}
```
Then the flag can be obtained by `imread_flags[imread_backend][flag]`. The argument `channel_order` can be removed then.

Reply:
By using the same set of flags, we can minimize the changes required to switch backends.

BTW the newly implemented rgb order for cv2 is efficient because it uses in-place operations.